### PR TITLE
Update Unity SDK docs

### LINKED ISSUE #69: Update Unity SDK docs
Migrate latest changes to the Unity SDK legacy docs to these docs. Note that the content structure may have changed significantly.

Changes from the following commits need to be addressed:
https://github.com/MetaMask/metamask-sdk-docs/commit/36b71caf75e92ba64d8e842e6759a4170dce9e5e
https://github.com/MetaMask/metamask-sdk-docs/commit/aadbaaab70e6e454b82f3f894dff1e378b4c79e0
https://github.com/MetaMask/metamask-sdk-docs/commit/bb33f6e32f455773d3c45210e30654de865bfe33

### PR DESCRIPTION
**Pull request by alexandratran**
Integrate latest changes to the legacy SDK docs, editing them as needed.

fixes #69 

Preview: https://metamask.github.io/mm-docs-v2/69-unity-sdk/wallet/how-to/use-sdk/unity/

### CODE REVIEW

**Comment by alexandratran** [comment]
@metamaskbot publish-preview

**Review by andreahaku — APPROVED** [review]
LGTM

File: wallet/how-to/use-sdk/unity.md [hunk 1]
```diff
@@ -25,20 +25,40 @@ The following video demonstrates how to install and use the MetaMask Unity SDK.
 
 ## Steps
 
-### 1. Install the SDK for Unity
+### 1. Configure build settings
 
-To install the module, first download the
-[Unity SDK Package](https://drive.google.com/u/0/uc?id=1ArTJvKIZXK5vkUOM3cgr0t0NspenWRU9&export=download).
+In order to build the SDK, configure the following settings in your Unity editor according to your
```
Comment (bgravenorst) [inline wallet/how-to/use-sdk/unity.md:30]:
```suggestion
To build the SDK, configure the following settings in your Unity editor according to your
```

File: wallet/how-to/use-sdk/unity.md [hunk 2]
```diff
@@ -224,3 +244,51 @@ You can use it to call any [provider API method](../../reference/provider-api.md
...
 | `Third Party Notices.md` | Third party notices                            |
+
+## FAQS
+
+<details>
+
+<summary>I can't find the SDK installation option.</summary>
+
+If you don't see the option to [install the SDK](#2-install-the-sdk-for-unity) in your Unity menu,
+make sure you're on the latest Unity version and that you have no red errors printed in your console.
```
Comment (bgravenorst) [inline wallet/how-to/use-sdk/unity.md:255]:
```suggestion
ensure you're on the latest Unity version and that you have no red errors printed in your console.
```

**Review by bgravenorst — APPROVED** [review]
LGTM

**Pull request merged**